comp/logs/agent/agent.go

```diff
@@ -176,6 +179,10 @@ func (a *agent) start(context.Context) error {
 	a.startPipeline()
 	a.log.Info("logs-agent started")
 
+	for _, scheduler := range a.schedulerProviders {
```

Comment:
**❓ question:** the code you removed from `cmd/agent/subcommands/run/command.go` only added one scheduler. Do we have plan to have more scheduler later or did you simply want this to be as generic as possible ?

Reply:
There is no plan to migrate other schedulers but if we will do it, there will be not need to update adscheduler.